batch-setup/make_meta_tiles.py

```diff
@@ -235,7 +242,7 @@ def __call__(self, parent):
         assert dz >= 0
         width = 1 << dz
 
-        size = 0
+        sizes = {}
```

Comment:
This might be easier to read. `{}` always looks like an empty `dict` to me, not a set.

```suggestion
        sizes = set()
```

Reply:
this is actually a dict! I need to track which tile has that size so I can combine them correctly at lower zooms

Reply:
Oh, well nevermind then. Maybe `dict()` then? 😄